Fw/Types/BasicTypes.hpp

```diff
@@ -104,56 +104,56 @@ typedef char CHAR;
 
 
 #ifndef I8_MAX
-#define I8_MAX (I8)(127)
+#define I8_MAX I8(127)
```

Comment:
I don't understand this syntax.  Is it described anywhere?

Reply:
"This cast expression is exactly equivalent to the corresponding C-style cast expression."  is what I found in the C++ documentation.  So, how is this better?

Reply:
Apparently all I managed was outsmarting the static analysis 🤦  I made the incorrect assumption that C++ would treat primitive type constructors/functional casts similar to assignment... thanks for bringing this up and making me dive into this more. Why C++ decided to make the behavior identical to a C style cast is a mystery to me.